src/app/main-overview/components/ControlPlaneDetails.vue

```diff
@@ -0,0 +1,209 @@
+  <div class="stack">
+    <KCard>
+      <template #body>
+        <div class="card-header">
+          <div class="card-title">
+            <h2>{{ t('main-overview.detail.health.title') }}</h2>
+          </div>
+        </div>
+
+        <div class="variable-columns">
+          <ResourceStatus
+            :total="store.getters['config/getMulticlusterStatus'] ? props.zoneOverviews.length : 1"
+            data-testid="zone-control-planes-status"
+          >
+            <template #icon>
+              <img src="@/assets/images/icon-location-on.svg?url">
+            </template>
+
+            <template #title>
+              {{ t('main-overview.detail.health.zone_control_planes') }}
+            </template>
+          </ResourceStatus>
+
+          <ResourceStatus
+            :total="meshInsight.meshesTotal"
+            data-testid="meshes-status"
+          >
+            <template #icon>
+              <img src="@/assets/images/icon-circles-ext.svg?url">
+            </template>
+
+            <template #title>
+              {{ t('main-overview.detail.health.meshes') }}
+            </template>
+          </ResourceStatus>
+
+          <ResourceStatus
+            :total="meshInsight.services.total"
+            data-testid="services-status"
+          >
+            <template #icon>
+              <img src="@/assets/images/icon-wifi-tethering.svg?url">
+            </template>
+
+            <template #title>
+              {{ t('main-overview.detail.health.services') }}
+            </template>
+          </ResourceStatus>
+
+          <ResourceStatus
+            :total="meshInsight.dataPlaneProxies.total"
+            data-testid="data-plane-proxies-status"
+          >
+            <template #icon>
+              <img src="@/assets/images/icon-wifi-tethering.svg?url">
+            </template>
+
+            <template #title>
+              {{ t('main-overview.detail.health.data_plane_proxies') }}
+            </template>
+          </ResourceStatus>
+        </div>
+      </template>
+    </KCard>
+
+    <div class="variable-columns">
+      <KCard v-if="store.getters['config/getMulticlusterStatus']">
+        <template #body>
+          <div class="card-header">
+            <div class="card-title">
+              <h2>{{ t('main-overview.detail.zone_control_planes.title') }}</h2>
+
+              <RouterLink :to="{ name: 'zone-cp-list-view' }">
+                {{ t('main-overview.detail.health.view_all') }}
+              </RouterLink>
+            </div>
+
+            <div
+              v-if="env('KUMA_ZONE_CREATION_FLOW') === 'enabled'"
+              class="card-actions"
+            >
+              <KButton
+                appearance="creation"
+                icon="plus"
+                :to="{ name: 'zone-create-view' }"
+              >
+                {{ t('zones.index.create') }}
+              </KButton>
+            </div>
+          </div>
+
+          <EmptyBlock v-if="props.zoneOverviews.length === 0">
```

Comment:
note: we're not doing `!== undefined` because we're dealing with an array and in other places we had objects